modin/experimental/engines/pandas_on_ray/sql.py

```diff
@@ -271,7 +271,7 @@ def query_put_bounders(query, partition_column, start, end):
 
 
 class InvalidArguments(Exception):
-    """Exception, that should be raised if invalid arguments combination was found."""
+    """Exception that should be raised if invalid arguments combination was found."""
 
     pass
```

Comment:
note that `pass` isn't needed here anymore, a docstring should be enough to mark a block :)

Reply:
Fixed.